config/default.yml

```diff
@@ -55,12 +55,12 @@ Sorbet/FalseSigil:
 
 Sorbet/ForbidIncludeConstLiteral:
   Description: 'Forbids include of non-literal constants.'
-  Enabled: true
+  Enabled: false
   VersionAdded: 0.2.0
 
 Sorbet/ForbidSuperclassConstLiteral:
   Description: 'Forbid superclasses which are non-literal constants.'
-  Enabled: true
+  Enabled: false
   VersionAdded: 0.2.0
```

Comment:
We haven't been using them so far, but can we also add a `VersionChanged: 0.5.0` for these two changes?

Reply:
I think it makes sense to do that in the `autocorrect` PR? Does this count as a change to the cop itself?

Reply:
According to https://github.com/rubocop-hq/rubocop/issues/5978, the defaults changing is also captured with `VersionChanged`